README.md

```diff
@@ -1,21 +1,34 @@
 # \<kwc-challenge-card\>
 
-To display a building challenge, and indicate whether the user has completed this or not.
+## Purpose:
+To display a building challenge, and indicate whether it is completed not.
 
-## Install the Polymer-CLI
+## Variants:
+* complete
 
-First, make sure you have the [Polymer CLI](https://www.npmjs.com/package/polymer-cli) installed. Then run `polymer serve` to serve your element locally.
+## Properties
+* challengeid (String): Unique challenge identifier. Currently the challenge slug. 
```

Comment:
To respect the naming convention of HTML attributes, can we make this property `challengeId`, that would translate to `challenge-id`